docs/source/troubleshooting/submit_debug_info.rst

```diff
@@ -81,3 +81,128 @@ and uploaded to StackStorm.
     2015-02-10 17:43:49,016  INFO - Collecting files...
     2015-02-10 17:43:49,770  INFO - Creating tarball...
     2015-02-10 17:43:49,912  INFO - Debug tarball successfully generated and can be reviewed at: /tmp/st2-debug-output-vagrant-ubuntu-trusty-64-2015-02-10-17:43:49.tar.gz
+
+
+Submitting debugging information to Plexxi
```

Comment:
Probably want to make this more generic and not talk about Plexxi. 
"Customizing the debug information gathered"